### DISCUSSION TITLE
Make the hat blocks cat blocks

### DISCUSSION THREAD

**Discussion by gitttyhub** [ScratchAddons/ScratchAddons#1422] [Ideas] [answered]
like the cat blocks from April fools

![image](https://user-images.githubusercontent.com/78053835/105917486-3834ca80-5fe7-11eb-950d-d5d94f2d5e0d.png)

**Comment by TheColaber** [comment, marked as answer]
Hey @gitttyhub we actually already have that addon! But we have hidden it! Do the Konomi code in the settings page, and it will appear!
